data-prepper-plugins/event-json-codecs/src/main/java/org/opensearch/dataprepper/plugins/codec/eventjson/EventJsonDefines.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.opensearch.dataprepper.plugins.codec.eventjson;
```

Comment:
The Java convention for packages with spaces is to use underscore. So you could rename to:

```
package org.opensearch.dataprepper.plugins.codec.event_json;
```